.yapfignore

```diff
@@ -0,0 +1 @@
+venv
```

Comment:
We can delete this file.

Reply:
Deleted: https://github.com/pfnet/optuna/pull/319/commits/5ebce4d2f7cc70d9e0949f1070d41f9db991d256